EIPS/eip-5656.md

```diff
@@ -0,0 +1,104 @@
+---
+eip: 5656
+title: Memory copying instruction
+description: An efficient EVM instruction for copying memory areas
+author: Alex Beregszaszi (@axic), Paul Dworzanski (@poemm), Jared Wasinger (@jwasinger), Casey Detrio (@cdetrio), Pawel Bylica (@chfast), Charles Cooper (@charles-cooper)
+discussions-to: https://ethereum-magicians.org/t/eip-5656-mcopy-instruction/10890
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-02-01
+---
+
+## Abstract
+
+Provide an efficient EVM instruction for copying memory areas.
```

Comment:
This should be (at a minimum) a full descriptive sentence describing a high-level overview 
of the solution that this EIP provides:
```suggestion
This EIP defines a new EVM opcode, `MCOPY`, for efficiently copying continuous sections of memory.
```

Reply:
Yep, good copywrite, this's much more readable. @Pandapip1

Reply:
This defines a new instruction. Not opcode which is just a number.

Reply:
Opcodes can be referred to either by number or by name. Feel free to correct me if I am wrong though.

Reply:
I also prefer the current status quo of the distinction between instruction and opcode.